testsuite/tests/parsetree/source.ml

```diff
@@ -7520,3 +7520,8 @@ type t = String.( t )
 
 (* Utf8 identifier *)
 let là = function ça -> ça
+
+let x = match 'é' with
+  | 'é' | 'œ' -> 'ᠫ'
+  | '𐂶' ->  'e'
+  | _ -> 'U+0504'
```

Comment:
It'd be good to have a test for the error when something that doesn't fit in a codepoint, like an emoji ?